docs/features/backup_and_restore.md

```diff
@@ -3,12 +3,190 @@ id: backup_and_restore
+such, the simplest recovery approach  is to re-deploy your database instance
+(e.g. your RDS cluster) and re-run the database data migration
+module to migrate Collections, Providers, etc, from a (possibly recovered
+DynamoDB instance) then begin active migration of Files and Granules records.
+
+For larger datastores this approach may not work
+due to migration time/other limitations.  In that case, utilizing an RDS recovery
+approach in conjunction with the Dynamo to RDS reconciliation tooling
+may be appropriate.
+
+#### Backup and Restore with AWS RDS
+
+##### Configuring Database Backups
+
+For AWS RDS Aurora database deployments, AWS provides a host of database
+backup/integrity options, including [PITR (Point In Time
+Recovery)](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PIT.html)
+based on automated database backups and replay of transaction logs.
+
+For further information on RDS backup procedures, see the [AWS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_CommonTasks.BackupRestore.html)
+
+##### Disaster Recovery
+
+To recover a Cumulus Postgres database in a disaster or data-loss scenario, you should perform the following steps:
+
+* If the Postgres database cluster exists/is still online, take it
+  offline/remove access.
+* If needed, recover the DynamoDB tables as noted in the
+  [DyanmoDb](./backup_and_restore#dynamodb) section of this document.
+* Redeploy a new database cluster from your backup, matching as closely as possible to the DynamoDB
+  restore time.   See [AWS's
+  PIT recovery
+  instructions](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PIT.html)
+  and [DB Snapshot recovery
+  instructions](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RestoreFromSnapshot.html),
+  or the examples below for more information.
+* Configure your Cumulus deployment to utilize the new database cluster and re-deploy
+* Run DynamoDB/RDS reconciliation tools and resolve any discrepancies.
+
+##### cumulus-rds-tf examples
+
+The following sections provide a walk through of a few recovery scenarios for the provided `cumulus-rds-tf`
+serverless module.
+
+***Point In Time Recovery***
+
+If you need recovery that exceeds the 1-day granularity of AWS's snapshots, you
+either must create and manually manage snapshots, or use Point In Time
+Recovery(PITR) if you still have the original cluster available.
+
+Unfortunately as terraform does not yet support RDS PITR (see:
+[github terraform-provider issue #5286](https://github.com/terraform-providers/terraform-provider-aws/issues/5286)),
+this requires a manual procedure.
+
+If you are using the `cumulus-rds-tf` module to deploy an RDS Aurora Serverless
+Postgres instance, the following procedure can be used to successfully spin up a duplicate
+cluster from backup in recovery scenarios:
+
+1. Halt all ingest and remove access to the database to prevent Core processes from
```

Comment:
Should we say how to remove access?

Reply:
Good point. 

We have a couple of options there -- we can go the DB route where you do something like set max connections to 0 and kick all active connections for the *database* they're using, or we can invalidate the access secret which should kill any *new* connections.

Reply:
c9c68fe12 - I think, though, we have a larger problem in that halting an event-driven/async system is by nature fraught.    Killing database access may stop workflows, but it won't stop rules from being run, or dynamo/elastic search from being used for reads.